pinax/stripe/models.py

```diff
@@ -197,10 +199,43 @@ class TransferChargeFee(models.Model):
     created_at = models.DateTimeField(default=timezone.now)
 
 
+class UserAccount(models.Model):
+    user = models.ForeignKey(settings.AUTH_USER_MODEL,
+                             related_name="user_accounts",
+                             related_query_name="user_account",
+                             on_delete=models.CASCADE)
+    account = models.ForeignKey("pinax_stripe.Account",
+                                related_name="user_accounts",
+                                related_query_name="user_account",
+                                on_delete=models.CASCADE)
+    customer = models.ForeignKey("pinax_stripe.Customer",
+                                 related_name="user_accounts",
+                                 related_query_name="user_account",
+                                 on_delete=models.CASCADE)
+
+    class Meta:
+        unique_together = ("user", "account")
+
+    def clean(self):
+        if not self.customer.stripe_account == self.account:
+            raise ValidationError(_("customer.stripe_account must be account."))
+        return super(UserAccount, self).clean()
+
+    def save(self, *args, **kwargs):
+        self.full_clean()
+        return super(UserAccount, self).save(*args, **kwargs)
```

Comment:
Not sure if we should override `save` here.
But then it seems to be the only way to enforce this validation?!

Reply:
Yep, according to django documentation. Validators are called by the Forms API. Since we don't use them, we have to `full_clean` ourselves.